### ISSUE TITLE
Add support for index templates

### ISSUE THREAD

**Issue by dadoonet** [dadoonet/fscrawler#643]
Instead of using index settings, we should always prefer using index templates.
We can just add a setting like `force_template: true` (`true` by default) which will always override any existing index template at startup.

The new file name for this can be `_template.json` and `_template_folder.json` for example.
Then we can move everything that we manually defined in `_settings.json` and `_settings_folder.json` to those new files.

That way people will still be able to either use an index template and/or overwrite the index template with whatever index settings.

When creating the job, we will use the default template definition available in FSCrawler and replace the `{{index_name}}` by the index name available in settings. Once generated this file won't be generated again.

**Comment by yossiarieli532** [comment]
I see this request is from almost two years ago. Is this feature live? 
I'm trying to write to an index alias but still get the same error.

Thanks!
